roles/etcd/templates/etcd-proxy-docker.service.j2

```diff
@@ -23,6 +23,8 @@ ExecReload={{ docker_bin_dir | default("/usr/bin") }}/docker restart {{ etcd_pro
 ExecStop={{ docker_bin_dir | default("/usr/bin") }}/docker stop {{ etcd_proxy_member_name | default("etcd-proxy") }}
 Restart=always
 RestartSec=15s
+OnStartupSec={{ etcd_start_delay }}
```

Comment:
I don't see why we need 3 timers. Let it fail the first time and restart after 15s as before. I thought we only needed to fix upstart provider